test_rclcpp/test/test_local_parameters.cpp

```diff
@@ -15,6 +15,7 @@
 #include <iostream>
 #include <stdexcept>
 #include <string>
+#include <cstdint>
```

Comment:
Already fixed this, the build is here http://ci.ros2.org/job/ci_windows/1611/

That line used to be `#include <stdint.h>` which caused the error